workflows/database_upsert.go

```diff
@@ -108,6 +110,14 @@ func (workflow *databaseWorkflow) databaseDeployer(namespace string, service *co
 			dbPassVersion, err := paramManager.ParamVersion(dbPassSSMParam)
 			if err != nil {
 				log.Warningf("Error with ParamVersion for DatabaseMasterPassword, assuming empty: %s", err)
+				answer, err := workflow.cliExtension.Prompt("Error retrieving DatabaseMasterPassword. Set a new DatabaseMasterPassword", false)
```

Comment:
This will cause issues for pipelines.  The first time the pipeline runs, the prompt will fail since it is running without a TTY attached.  Maybe just check if there is a TTY before prompting, othewise assume it's ok?

Check out something like what was done for prompting for github password on `mu pipeline up`...makes testing much easier: https://github.com/stelligent/mu/blob/develop/cli/pipelines.go#L73

Reply:
Ah, I wasn't aware this was called without a TTY attached at any point. I will make sure to handle that.

I did see how you handled pipeline up. I avoided that pattern because of where the prompt is happening (deep in the upsert command). But, maybe I need to consider a larger refactor of this method to move the param checking outside of this method entirely and attach it somewhere else. I'll revisit what I did here and see if I can simplify or abstract it